src/App.tsx

```diff
@@ -1,24 +1,71 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isVisible: boolean,
+  clockName: number,
 };
 
+class App extends React.Component<{}, State> {
+  state = {
+    isVisible: true,
+    clockName: 0,
+  };
+
+  clockHide = () => {
+    this.setState({ isVisible: false });
+  };
+
+  clockShow = () => {
+    this.setState({ isVisible: true });
+  };
+
+  ClockName = () => {
```

Comment:
в JS мы называем функции и переменные camelCase
функции должна описывать действие в своем названии

```suggestion
  changeClockName = () => {
```